runelite-client/src/main/java/net/runelite/client/plugins/statusbars/StatusBarsOverlay.java

```diff
@@ -129,23 +144,35 @@ public Dimension render(Graphics2D g)
 		final Point offsetLeft = curViewport.getOffsetLeft();
 		final Point offsetRight = curViewport.getOffsetRight();
 		final Point location = curWidget.getCanvasLocation();
-		final int height, offsetHealthX, offsetHealthY, offsetPrayerX, offsetPrayerY;
+		final boolean resizedBottomMode = curViewport == Viewport.RESIZED_BOTTOM;
+
+		final int height, offsetHealthX, offsetHealthY, offsetPrayerX, offsetPrayerY, offsetEnergyX, offsetEnergyY;
 
-		if (curViewport == Viewport.RESIZED_BOTTOM)
+		if (resizedBottomMode)
 		{
 			height = RESIZED_BOTTOM_HEIGHT;
+
 			offsetHealthX = (location.getX() + RESIZED_BOTTOM_OFFSET_X - offsetLeft.getX());
 			offsetHealthY = (location.getY() - RESIZED_BOTTOM_OFFSET_Y - offsetRight.getY());
+
 			offsetPrayerX = (location.getX() + RESIZED_BOTTOM_OFFSET_X - offsetRight.getX());
 			offsetPrayerY = (location.getY() - RESIZED_BOTTOM_OFFSET_Y - offsetRight.getY());
+
+			offsetEnergyX = (location.getX() - RESIZED_BOTTOM_PADDING_X - (offsetRight.getX() * 2));
+			offsetEnergyY = (location.getY() - RESIZED_BOTTOM_OFFSET_Y - offsetRight.getY());
 		}
 		else
 		{
 			height = HEIGHT;
+
 			offsetHealthX = (location.getX() - offsetLeft.getX());
 			offsetHealthY = (location.getY() - offsetLeft.getY());
+
 			offsetPrayerX = (location.getX() - offsetRight.getX()) + curWidget.getWidth();
 			offsetPrayerY = (location.getY() - offsetRight.getY());
+
+			offsetEnergyX = 0;
+			offsetEnergyY = 0;
```

Comment:
These are never used right?

Reply:
they have to be initialised, unfortunately. i've moved it outside of the conditionals.